libkbfs/disk_block_cache_test.go

```diff
@@ -53,15 +53,16 @@ func newDiskBlockCacheStandardForTest(config *testDiskBlockCacheConfig,
 	if err != nil {
 		return nil, err
 	}
-	limiter, err := newBackpressureDiskLimiterWithFunctions(
-		config.MakeLogger(""), 0.5, 0.95, 0.25, 0.25,
-		testDiskBlockCacheMaxBytes, maxFiles, time.Second,
-		defaultDoDelay, func() (int64, int64, error) {
-			// hackity hackeroni
-			freeBytes := maxBytes - int64(cache.currBytes)
-			return freeBytes, maxFiles, nil
-		})
-	// Sooooo hacky :(
+	if limiter == nil {
+		limiter, err = newBackpressureDiskLimiterWithFunctions(
+			config.MakeLogger(""), 0.5, 0.95, 0.25, 0.25,
+			testDiskBlockCacheMaxBytes, maxFiles, time.Second,
+			defaultDoDelay, func() (int64, int64, error) {
+				// hackity hackeroni
```

Comment:
Maybe a slightly more detailed comment? Something like: `Hack: simulate the disk cache taking up space.`